airflow/www/security.py

```diff
@@ -520,18 +560,23 @@ def merge_pv(perm, view_menu):
     def update_admin_perm_view(self):
         """
         Admin should has all the permission-views, except the dag views.
-        because Admin have already have all_dags permission.
+        because Admin have already have Dag permission.
         Add the missing ones to the table for admin.
 
         :return: None.
         """
-        all_dag_view = self.find_view_menu('all_dags')
-        dag_perm_ids = [self.find_permission('can_dag_edit').id, self.find_permission('can_dag_read').id]
+        all_dag_view = self.find_view_menu('Dag')
+        pvs = (
+            self.get_session.query(sqla_models.ViewMenu)
+            .filter(sqla_models.ViewMenu.name.like(f"{DAG_PREFIX}%"))
+            .all()
+        )
+        pv_ids = [pv.id for pv in pvs]
```

Comment:
```suggestion
        pv_ids = (
            self.get_session.query(sqla_models.ViewMenu.id)
            .filter(sqla_models.ViewMenu.name.like(f"{DAG_PREFIX}%"))
            .all()
        )
```
I think?

(Though that might yield `[[id1],[id2]]`)

Reply:
This yields `((id1,), (id2,), (id3,))`

Reply:
(not saying this is a good idea, or even clear to read) but:

```python shell
>>> (x,) = zip(*((1,),(2,)))                                                                                                                                                                                                                                                                                                                                            
>>> x                                                                                                                                                                                                                                                                                                                                                                   
(1, 2)
```

Reply:
Interesting. Good to have this in my back pocket. Although I agree that for now, it probably reduces readability.